2023/README.md

```diff
@@ -1,20 +1,276 @@
+
+[https://html.spec.whatwg.org/multipage/forms.html#the-form-element](https://html.spec.whatwg.org/multipage/forms.html#the-form-element)
+
+
+### Grid
+
+CSS Grid Layout excels at dividing a page into major regions or defining the relationship in terms of size, position, and layer, between parts of a control built from HTML primitives.
+
+[https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)
+
+[https://drafts.csswg.org/css-grid/](https://drafts.csswg.org/css-grid/)
+
+[https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-grid](https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-grid)
+
+
+### :has()
+
+With a :host() selector referencing the root of an element with a shadow root, it holds that :host(:has(...) should reference light DOM children of that root. Some browsers acknowledge this truth (Safari), others don't (Chrome), others have yet to ship :has() support (Firefox).
+
+[https://developer.mozilla.org/en-US/docs/Web/CSS/:has](https://developer.mozilla.org/en-US/docs/Web/CSS/:has)
+
+[https://w3c.github.io/csswg-drafts/selectors/#relational](https://w3c.github.io/csswg-drafts/selectors/#relational)
+
+[https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-has](https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-has)
+
+
+### Inert
+
+The HTMLElement property inert reflects the value of the element's inert attribute. It is a boolean value that, when present, makes the browser "ignore" user input events for the element, including focus events and events from assistive technologies. The browser may also ignore page search and text selection in the element. This can be useful when building UIs such as modals where you would want to "trap" the focus inside the modal when it's visible.
+
+[https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert)
+
+[https://html.spec.whatwg.org/multipage/interaction.html#the-inert-attribute](https://html.spec.whatwg.org/multipage/interaction.html#the-inert-attribute)
+
+[https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-inert](https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-inert)
+
+
+### Masking in CSS
+
+CSS Masking is a CSS module that defines means, including masking and clipping, for partially or fully hiding portions of visual elements.
```

Comment:
With "math" being on the screen close by, I took "mean" to mean the mathematical concept at first. Simpler language:

```suggestion
CSS Masking allows for partially or fully hiding portions of visual elements, using masking and clipping.
```